client-core/src/service/root_hash_service.rs

```diff
@@ -52,7 +52,7 @@ where
         let merkle_tree = MerkleTree::new(combinations);
         let root_hash = merkle_tree.root_hash();
 
-        let multi_sig_address = MultiSigAddress { m, n, merkle_tree };
+        let multi_sig_address = MultiSigAddress {m: m as u64, n:n as u64, merkle_tree };
```

Comment:
Why not take `u64` in arguments instead of casting `usize` to `u64`?

Reply:
done